packages/genesys-spark-components/src/components/stable/gux-copy-to-clipboard/gux-copy-to-clipboard.scss

```diff
@@ -42,6 +44,7 @@ button {
     gux-icon[icon-name='copy'] {
       flex-shrink: 0;
       width: var(--gse-ui-icon-size-sm);
+      min-width: 22px;
```

Comment:
22px was a size that I found in playing around with the icon.  Otherwise, it is fairly arbitrary. 

This is important because I've found that the icon shrinks far too small in the event the content is too big for the space its in, and the container wraps the text.

Reply:
It keeps the icon from shrinking too small.  I've had the copy icon shrink to the size of a dot if the text for copy-to-clipboard is too big, and the container it is in wraps text. 

`min-width` just defines a minimum size for the icon so that does not happen.  Looking through the changelog, I saw this was an issue that was attempted to be addressed in a 4.0.0 beta. 
https://github.com/MyPureCloud/genesys-spark/commit/2b65c9278268aca2163c8cc10a969e8ef0658660

Reply:
Hi @harter-genesys 

Can you send on some more info about how to recreate the shrinking icon issue? I've tried altering the font-size and content of the main section of the copy-to-clipboard but I couldn't replicate it. 

Thanks,
Gillian

Reply:
I'm having troubles directly replicating without using the `fixtable` package our app is using for tables.  It appears the table-layout is what is causing the shrinking to occur (that in conjunction with something else).  I'm having a hard time reproducing this in the webcomponent playground.  If I'm unable to get an example, I am fine to remove this bit.
With the `table-layout` set to `fixed`:
<img width="204" alt="image" src="https://github.com/user-attachments/assets/f83db8e2-7a0a-46fb-8f2d-920074884869">
With `table-layout` unset:
<img width="296" alt="image" src="https://github.com/user-attachments/assets/02b4c8d5-6af9-42fe-bd23-9c066b5dde0d">

Here are the styles for the top level table element, but like I said, using those styles in the playground don't appear to be replicating the issue:
```html
<table class="table table-hover" style="margin: 0px;padding: 0px;border-width: 0px;border-style: initial;border-color: initial;table-layout: fixed;">
```

Reply:
@321gillian I figured it out.  For some reason, the styles for `gux-icon` are not being applied in my usage.  That `flex-shrink: 0` works to resolve the issue pasted in the screenshot.  I'm investigating why these styles are not present

Reply:
it looks like it was a bug fixed a couple weeks ago in 4.64.4 (right after I opened this ticket).  The selector in scss was incorrect and the styles were not being applied.  
https://github.com/MyPureCloud/genesys-spark/commit/741f6792b498d6e907b3d47beb903f1f64fdf7e4